app/src/features/Mappings/Create/CreateMapping.tsx

```diff
@@ -48,34 +44,26 @@ import {
   resourceAdded,
 } from "../resourceSlice";
 
-const FOOTER_HEIGHT = 150;
-
 const useStyles = makeStyles((theme) => ({
   footerContainer: {
-    height: FOOTER_HEIGHT,
+    height: theme.mixins.footer.height,
     display: "flex",
     alignItems: "center",
     justifyContent: "center",
     boxShadow: `0px 0px 10px ${theme.palette.divider}`,
     width: "100%",
   },
   rootContainer: {
-    height: `calc(100vh - ${FOOTER_HEIGHT + theme.spacing(10)}px)`,
+    height: `calc(100vh - ${
+      Number(theme.mixins.footer.height) + Number(theme.mixins.appbar.height)
+    }px)`,
```

Comment:
```suggestion
    height: `calc(100vh - ${2 * Number(theme.mixins.footer.height)}px)`,
```

Reply:
No it's not the same size ! It's footer and appbar !

Reply:
No it's not the same size ! It's footer and appbar !